packages/syft/src/syft/util/trace_decorator.py

```diff
@@ -39,9 +44,6 @@ def set_default_attributes(cls, attributes: dict[str, str] | None = None) -> Non
                 cls.default_attributes[att] = attributes[att]
 
 
-T = TypeVar("T", bound=Callable | type)
-
-
 def instrument(
```

Comment:
While Testing I found that, we do not capture blob storage , as we upload them directly.
I think it would be good to enable otel at the ingress controller.

For now, we could enable at the internal traefik proxy  level.
Traefik has native support: https://traefik.io/blog/monitor-your-production-at-a-glance-with-traefik-3-0-and-opentelemetry/

when we remove traefik as internal proxy, we could enable at ingress controller when we switch to nginx.

Reply:
Enabling at ingress controller, would additionally give a full view of the traces .

Reply:
Locally, that's taken care of by signoz otel collector installed in the cluster `default` namespace.
On cloud, this is already taken care of by default.

Reply:
> While Testing I found that, we do not capture blob storage , as we upload them directly.
isn't botocore instrumentation doing that for us?